tools/validate_configs/test_configs/2filestore-4instances.yaml

```diff
@@ -53,13 +53,10 @@ deployment_groups:
       - type: shell
         content: $(homefs.install_nfs_client)
         destination: "install-nfs.sh"
```

Comment:
Now that you added the `use: homefs` you can also remove the `install-nfs.sh` runner (and this whole startup script block).